dataset/src/test/scala/frameless/TypedDatasetSuite.scala

```diff
@@ -6,11 +6,7 @@ import org.scalactic.anyvals.PosZInt
 import org.scalatest.FunSuite
 import org.scalatest.prop.Checkers
 
```

Comment:
I'm unbiased about the original motivation for `Job`, but if it's going to be there, it ought to have these operations IMHO.